watertap/property_models/activated_sludge/asm1_properties.py

```diff
@@ -264,7 +265,7 @@ def build(self):
         self.temperature = pyo.Var(
             domain=pyo.NonNegativeReals,
             initialize=298.15,
-            bounds=(298.15, 323.15),
+            bounds=(293.15, 323.15),
```

Comment:
I think the lower bound for temperature is still a little high. Even though it is an exothermic reaction, the initial environmental temperature can be low in winter, so probably set to 273.15?

Reply:
Activated sludge processes don't work at low temperatures, so I think this is probably OK - either way that is something the user should update on a c